### PR TITLE
Add conformance test for KeyUsage bits

### PR DESCRIPTION
**Pull request by ties**
Add conformance test cases for KeyUsage bits in CA certificates.

Could have been a parameterised test with JUnit Jupiter.

### CODE REVIEW

**Review by wibisono — APPROVED** [review]

**Comment by wibisono** [comment]
Just a remark, there is [badCertNoKeyUsage](https://github.com/RIPE-NCC/rpki-commons/blob/master/src/test/resources/conformance/root/badCertNoKeyUsage.cer) which seems related. 
But since it is not specifically about checking bits, maybe it does not belong here.

**Comment by ties** [comment]
Now I see the mail in my mailbox I realise that I did not comment on the pr
(only in de code): seems like we did not check this.

That causes the tests to fail.

File: src/test/java/net/ripe/rpki/commons/validation/interop/BBNConformanceTest.java [hunk 1]
```diff
@@ -68,6 +69,8 @@ public void shouldParseAllObjects() throws IOException {
                     new ManifestCmsParser().parse(result, encoded);
                 } else if (file.getName().endsWith("roa")) {
                     new RoaCmsParser().parse(result, encoded);
+                } else if (file.getName().endsWith("gbr")) {
+                    new GhostbustersCmsParser().parse(result, encoded);
```
Comment (erikrozendaal) [inline src/test/java/net/ripe/rpki/commons/validation/interop/BBNConformanceTest.java:73]: This function should really use `CertificateRepositoryObjectFactory.createCertificateRepositoryObject`, which already contains the equivalent conditional based on file extensions.

**Comment by ties** [comment]
We do part of the validation of KeyUsage in `X509CertificateParentChildValidator` (which is not on this path). That implementation, however, does not check if the allowed bits are the only bits present.

**Comment by ties** [comment]
It now passes the conformance test. However, two serialisers are now broken.

**Pull request merged**